sdk/zklogin/src/nonce.ts

```diff
@@ -3,14 +3,16 @@
 
 import { base64url } from 'jose';
 import { toBigIntBE } from 'bigint-buffer';
+import { randomBytes } from '@noble/hashes/utils';
 import { PublicKey } from '@mysten/sui.js/cryptography';
 import { poseidonHash } from './poseidon.js';
 import { toBufferBE } from './utils.js';
 
 const NONCE_LENGTH = 27;
 
 export function generateRandomness() {
-	return toBigIntBE(Buffer.from(crypto.getRandomValues(new Uint8Array(16))));
+	// Once Node 20 enters LTS, we can just use crypto.getRandomValues(new Uint8Array(16)), but until then this improves compatibility:
```

Comment:
wasn't `crypto.getRandomValues` available starting in Node.js v15 (https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues)?

Reply:
Yeah but the `crypto` global isn't available until node 20, so we'd need to import `crypto` with breaks web compatibility. The package we use basically is a wrapper over this that imports `crypto` in node environments and uses web crypto in non-node environments. It just avoids us needing to do that work.